projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -171,6 +155,46 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     @ViewChild(IgxForOfDirective, { read: IgxForOfDirective, static: true })
     protected virtDir: IgxForOfDirective<any>;
 
+    /**
+     * Set the combo's overlaySettings that control how the list of items is displayed.
+     *
+     * ```html
+     * <igx-combo [overlaySettings] = "customOverlaySettings"></igx-combo>
+     * ```
+     *
+     * ```typescript
+     *  const customSettings = { positionStrategy: { settings: { target: myTarget } } };
+     *  combo.overlaySettings = customSettings;
+     * ```
+     */
+    @Input()
+    public set overlaySettings(val: OverlaySettings) {
+        if (val) {
+            const newSettings = val;
```

Comment:
Not sure what this is about. This is still by reference, overriding the property below still affects the user-provided object.